adafruit_bno055.py is failing pylint

[See Actions run](https://github.com/adafruit/Adafruit_CircuitPython_BNO055/runs/1328727892?check_suite_focus=true)
```
************* Module adafruit_bno055
adafruit_bno055.py:137:8: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
adafruit_bno055.py:141:17: R1725: Consider using Python 3 style super() without arguments (super-with-arguments)
```

Here are the 2 methods that are failing:
[Method 1](https://github.com/adafruit/Adafruit_CircuitPython_BNO055/blob/7030883ab28fecba2351443e7139200ac49d0135/adafruit_bno055.py#L136-L138)
```
136    def __init__(self, register_address, struct_format, scale):
137        super(_ScaledReadOnlyStruct, self).__init__(register_address, struct_format)
138        self.scale = scale
```

[Method 2](https://github.com/adafruit/Adafruit_CircuitPython_BNO055/blob/7030883ab28fecba2351443e7139200ac49d0135/adafruit_bno055.py#L140-L142)
```
140    def __get__(self, obj, objtype=None):
141        result = super(_ScaledReadOnlyStruct, self).__get__(obj, objtype)
142        return tuple(self.scale * v for v in result)
```

Should I remove the arguments and then try getting it to work that way or should I just add a pylint disable?

--- a/adafruit_bno055.py
+++ b/adafruit_bno055.py
@@ -134,11 +134,11 @@ _ID_REGISTER = const(0x00)
 
 class _ScaledReadOnlyStruct(Struct):  # pylint: disable=too-few-public-methods
     def __init__(self, register_address, struct_format, scale):
-        super(_ScaledReadOnlyStruct, self).__init__(register_address, struct_format)
+        super().__init__(register_address, struct_format)
         self.scale = scale
 
     def __get__(self, obj, objtype=None):
-        result = super(_ScaledReadOnlyStruct, self).__get__(obj, objtype)
+        result = super().__get__(obj, objtype)
         return tuple(self.scale * v for v in result)
 
     def __set__(self, obj, value):

